zproject/prod_settings_template.py

```diff
@@ -548,13 +548,13 @@
 # their email address is "sam@example.com", set LDAP_APPEND_DOMAIN to
 # "example.com".  Otherwise, leave LDAP_APPEND_DOMAIN=None and set
 # AUTH_LDAP_REVERSE_EMAIL_SEARCH and AUTH_LDAP_USERNAME_ATTR below.
-LDAP_APPEND_DOMAIN = None  # type: Optional[str]
+LDAP_APPEND_DOMAIN: Optional[str] = None
```

Comment:
I'm not sure about the changes in this file; one of the benefits of the old system was that folks who didn't know Python were less likely to be confused in trying to fill in the template, and I worry that this `:` syntax may throw off sysadmins.  I suppose one option is we could run something more complex than `cp` when installing the template that stripe type annotations?

Reply:
What if we move the `None` values to `default_settings` and just leave them commented here?

Reply:
Yeah, if we make them commented here with e.g. `= "example.com"`, that would be a fine solution.  And backwards-compatibility would be correct; if you have an old `settings.py` file, you'll end up with None just the same.  It's likely worth including a comment in the `default_settigns.py` entry that we can't change the semantics of `None` because old `settings.py` files hardcode setting this to None.

Reply:
A commented `= None` should serve both purposes.

Reply:
I documented the purpose of `None` in the `SSO_APPEND_DOMAIN` comment, which was the only one that didn’t already document this.